doc/source/templates/04_finetuning_llms_with_deepspeed/finetune_hf_llm.py

```diff
@@ -9,7 +9,7 @@
 import pandas as pd
 from pathlib import Path
 import torch.nn as nn
-from ray import tune
+# from ray import tune
```

Comment:
We reference `tune` in a comment later in this template, so I wasn't sure if we should remove it entirely:

```
            # Turn off syncing artifact as as of 2.6 it introduces a resource
            # contention between checkpoint syncronizer and artifact syncronizer that
            # can sometimes result in failed checkpoint syncing
            # sync_config=tune.SyncConfig(sync_artifacts=False),
```